### PR TITLE
Adding default .github/

### PR DESCRIPTION
**Pull request by ryscheng**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .github/ is restricted
+.github/ @ryscheng
```
Comment (nhynes) [inline .github/CODEOWNERS]: this should be a team--not a (particularly busy) person

Comment (ravenac95) [reply .github/CODEOWNERS]: @nhynes teams wasn't working for whatever reason. will have to figure that out later

Comment (nhynes) [reply .github/CODEOWNERS]: can we just not impose this change then? it's like granting "emergency powers" in wartime and having them never be retracted

Comment (ravenac95) [reply .github/CODEOWNERS]: actually we figured out the problem 😄 we think... stand by

**Pull request merged**